src/components/VoteCell.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+export default function VoteCell(props) {
+  const { index, person, vote } = props;
+  const { id, name, voteCount } = person;
+
+  const voteToCandidate = () => {
```

Comment:
이벤트 핸들러 네이밍에는 보통 `handle`을 접두사로 붙여서 사용하므로 이렇게 네이밍해도 좋을 것 같습니당
```suggestion
  const handleVoteCount= () => {
```